app/views/patients/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<div>
+
+  <% if !flash[:message].blank? %>
+    <div>
+      <%= flash[:message] %>
+    </div>
+  <% end %>
+  <h2>Patients</h2>
+
+  <%= link_to("Add New Patient", new_patient_path, :class => 'action new') %>
+
+  <table>
+    <tr>
+      <th>#</th>
+      <th>First Name</th>
+      <th>Last Name</th>
+      <th>Actions</th>
+    </tr>
+    <% @patients.each do |patient| %>
+    <tr>
+      <td><%= patient.id %></td>
+      <td><%= patient.first_name %></td>
+      <td><%= patient.last_name%></td>
+      <td class="actions">
+        <%= link_to("Show", patient_path(patient), :class => 'action show') %>
+        <%= link_to("Edit", edit_patient_path(patient), :class => 'action edit') %>
+        <%= link_to("Delete", delete_patient_path(patient), :class => 'action delete') %>
+      </td>
+    </tr>
+    <% end %>
+  </table>
+</div>
```

Comment:
bad file ending. FYI the whole team likes to use VIM as a text editor. It has the positive feature of never leaving these bad file endings. Check out https://vim-adventures.com for some fun vim practice. We can buy you a license if you're interested.